services/web/client/source/class/qxapp/data/model/Project.js

```diff
@@ -0,0 +1,60 @@
+qx.Class.define("qxapp.data.model.Project",
+  {
+    extend: qx.core.Object,
+
+    properties: {
+      id: {
+        check: "String",
+        event: "changeId",
+        nullable: true
+      },
+
+      name: {
+        check: "String",
+        init: "Unnamed"
+      },
+
+      description: {
+        check: "String",
+        nullable: true
+      },
+
+      thumbnail: {
+        check: "String",
+        nullable: true
+      },
+
+      createdOn: {
+        check: "Date",
+        nullable: true
+      }
+
+    },
+
+    members: {
+
+      toString: function() {
+        // return qx.dev.Debug.debugProperties(this, 3, false, 2);
+        let newLine = "\r\n";
```

Comment:
windows newline ???

Reply:
mmhhh now that you mention it ...
This was a trial that I shamefully c&p-ed it from the code of [qx.dev.Debug](https://github.com/qooxdoo/qooxdoo/blob/1c53f45/framework/source/class/qx/dev/Debug.js#L307) and since I am so used to windows newline I simply missed it :)